src/llmcompressor/pytorch/model_load/helpers.py

```diff
@@ -87,22 +86,6 @@ def save_model_and_recipe(
     copy_python_files_from_model_cache(model, save_path)
 
 
-def fallback_to_cpu(device: str) -> str:
```

Comment:
Add a deprecation warning for logic change.

Reply:
Done